File: Atcoder/Ruby/126.ABC/B.rb
kes = gets.chomp

evis = kes[0,2]
jap = kes[-2,2]

if(evis >'00' && evis <'13' && jap >'00' && jap <'13')
  puts "AMBIGUOUS"
elsif (evis >'00' && evis <'13' )
  puts "MMYY"
elsif (jap >'00' && jap <'13')
  puts "YYMM"
else
  puts "NA"
end  

File: Atcoder/PHP/126.ABC/B.php
<?php
fscanf(STDIN, "%s", $kes);

$evis = substr($kes, 0, 2);
$jap = substr($kes, -2, 2);

$a = '03';

if ($evis > '00' && $evis < '13' && $jap > '00' && $jap < '13') {
    echo "AMBIGUOUS";
} elseif ($evis > '00' && $evis < '13') {
    echo "MMYY";
} elseif ($jap > '00' && $jap < '13') {
    echo "YYMM";
} else {
    echo "NA";
}